barista-node/package.json

```diff
@@ -5,21 +5,18 @@
   "main": "main.js",
   "scripts": {
     "start": "node .",
-    "lint": "eslint ."
+    "lint": "eslint --ignore-path .gitignore . --fix"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "chalk": "^4.1.0",
-    "kafkajs": "^1.14.0"
+    "dotenv": "^8.2.0",
+    "node-rdkafka": "^2.9.1",
+    "pino": "^6.7.0",
+    "pino-pretty": "^4.3.0"
   },
   "devDependencies": {
-    "eslint": "^7.11.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.13.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-prettier": "^3.1.4",
-    "prettier": "^2.1.2"
+    "eslint-config-airbnb-standard": "^3.1.0"
```

Comment:
Can we get this to be the same as the other nodeshift modules?  We are using semi-standard there

https://github.com/nodeshift/nodeshift/blob/master/package.json#L63
https://github.com/nodeshift/nodeshift/blob/master/.eslintrc.json